lib/lotus/commands/generate/action.rb

```diff
@@ -0,0 +1,235 @@
+        end
+
+        # @since x.x.x
+        # @api private
+        def route_url
+          options.fetch(:url, "/#{ @controller_name.downcase }")
+        end
+
+        # @since x.x.x
+        # @api private
+        def route_endpoint
+          "#{ @controller_name }#{ROUTE_ENDPOINT_SEPARATOR}#{ @action_name }".downcase
+        end
+
+        # @since x.x.x
+        # @api private
+        def known_application_names
+          Dir.glob(applications_path.join('/*')).map do |name|
+            File.basename(name)
+          end
+        end
+
+        # @since x.x.x
+        # @api private
+        def assert_controller_name!
+          if @controller_name.nil?
+            raise ArgumentError.new("Unknown controller, please add controllers name with this syntax controller_name#action_name")
+          end
+        end
+
+        # @since x.x.x
+        # @api private
+        def assert_action_name!
+          if @action_name.nil? || @action_name.strip == ''
+            raise ArgumentError.new("Unknown action, please add actions name with this syntax controller_name#action_name")
+          end
+        end
+
+        # @since x.x.x
+        # @api private
+        def assert_application_name!
+          return if !environment.container?
+          if @application_name.nil? || @application_name.strip == '' || !application_path.exist?
+            existing_apps = known_application_names.join('/')
+            raise ArgumentError.new("Unknown app: `#{ @application_name.downcase }'. Please specify one of (#{ existing_apps }) as application name")
+          end
+        end
+
+        # @since x.x.x
+        # @api private
+        def assert_http_method!
+          if !Lotus::Routing::Route::VALID_HTTP_VERBS.include?(http_method.upcase)
+            existing_http_methods = Lotus::Routing::Route::VALID_HTTP_VERBS
+            raise ArgumentError.new("Unknown HTTP method '#{http_method}', please use one of #{ existing_http_methods.join('/') }.")
+          end
+        end
+
+        def routes_path
+          if environment.container?
+            application_path.join('config', 'routes.rb')
+          else
+            application_path.join('..', 'config', 'routes.rb')
+          end
+        end
+
+        # The directory of the application
+        # ./app for 'app' architecture
+        # ./apps/APPLICATION_NAME for 'container'
+        def application_path
+          if environment.container?
+            applications_path.join(@application_name.downcase)
+          else
+            Pathname.new('app')
+          end
+        end
+
+        # The parent dir of the application directory.
+        def applications_path
+          Pathname.new('apps')
+        end
+
+        def view_path
+          application_path.join('views', @controller_name.downcase, "#{@action_name.downcase}.rb")
+        end
+
+        def view_spec_path
+          spec_root.join('views', @controller_name.downcase, "#{@action_name}_spec.rb")
```

Comment:
@t4deu thanks for offering help to @pascalbetz 

I am keen to get to this done before 23rd September release